src/awx_plugins/interfaces/_temporary_private_container_api.py

```diff
@@ -39,7 +40,10 @@ def get_incontainer_path(
     if not os.path.isabs(private_data_dir):
         raise RuntimeError('The private_data_dir path must be absolute')
 
-    container_root = pathlib.Path(CONTAINER_ROOT)
+    if not container_root:
+        container_root = CONTAINER_ROOT
+
+    container_root = pathlib.Path(container_root)
```

Comment:
(nit) this could be shorter
```suggestion
    container_root = pathlib.Path(container_root or CONTAINER_ROOT)
```